homeassistant/components/voip/devices.py

```diff
@@ -6,13 +6,22 @@
 
 from voip_utils import CallInfo
 
+from homeassistant.backports.enum import StrEnum
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import Event, HomeAssistant, callback
 from homeassistant.helpers import device_registry as dr, entity_registry as er
 
 from .const import DOMAIN
 
 
+class VadSensitivity(StrEnum):
```

Comment:
This should live in `assist_pipeline.vad` and we should make sure it's consistent across any integration that uses VAD (so also ESPHome).